test/image-blueprints/layer5-bootc/group2/cos9-bootc-source-flannel.containerfile

```diff
@@ -0,0 +1,23 @@
+FROM localhost/cos9-bootc-source:latest
+
+# Build arguments
+ARG REPO_CONFIG_SCRIPT=/tmp/microshift_repo_config.sh
+ARG USHIFT_RPM_REPO_NAME=microshift-local
+ARG USHIFT_RPM_REPO_PATH=/tmp/$USHIFT_RPM_REPO_NAME
+
+# Copy the repository configuration script
+COPY --chmod=755 ./bootc-images/microshift_repo_config.sh ${REPO_CONFIG_SCRIPT}
```

Comment:
This will no longer work as we deprecated the repo configuration script.
Please, use [this container file](https://github.com/ggiguash/microshift/blob/main/test/image-blueprints/layer5-bootc/group2/cos9-bootc-source-optionals.containerfile) as a reference on how to configure repos now.

Reply:
@ggiguash I need help around it, how I can verify changes locally? Also https://github.com/ggiguash/microshift/blob/main/test/README.md#blueprint-customization is this still correct or now it is `build_bootc_images.sh` ? I also need to understand how I can pass a env `WITH_FLANNEL=1`  to create flannel rpm.

Reply:
That section is missing the bootc image build instructions. I opened [USHIFT-4566](https://issues.redhat.com/browse/USHIFT-4566) to add the docs.

In the meantime, you need to run the following command to test the builds.
```
./test/bin/build_bootc_images.sh -l ./test/image-blueprints/layer5-bootc/ -b containerfile
```